pkg/analyzer/analyzer.go

```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/fatih/color"
 	"github.com/k8sgpt-ai/k8sgpt/pkg/common"
 	"github.com/k8sgpt-ai/k8sgpt/pkg/integration"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	AnalyzerErrorsMetric = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
I think it would make more sense to make this a counter (promauto.NewCounterVec)? 
```suggestion
	AnalyzerErrorsMetric = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Reply:
I'm not sure because this metric can decrease over time. What do you think ?

Reply:
I agree that having a gauge means we can do things like `rate[5m](AnalyzerErrors)` etc